tests/layer_tests/pytorch_tests/test_squeeze.py

```diff
@@ -29,9 +29,17 @@ def forward(self, x):
 
         return aten_squeeze(dim), ref_net, "aten::squeeze"
 
-    @pytest.mark.parametrize("dim", [-2, -1, 0, None, [-1, -2], 1, 2])
+    @pytest.mark.parametrize("dim", [-2, -3, 0, 1, [0, 1], None])
     @pytest.mark.nightly
     @pytest.mark.precommit
     def test_squeeze(self, dim, ie_device, precision, ir_version):
         # Dynamic shapes are introducing dynamic rank, with is not suppoerted by Squeeze operation.
         self._test(*self.create_model(dim), ie_device, precision, ir_version, dynamic_shapes=False)
+
+    @pytest.mark.xfail(reason='OpenVINO squeeze does not support axis size != 1')
```

Comment:
That is incorrect description. OV support axis size > 1, it doesn't support the axis -1 or 2 because dimension is not equal to 1.

Reply:
updated